lms/djangoapps/gradebook/management/commands/tests/test_generate_gradebook_entries.py

```diff
@@ -27,7 +33,7 @@ class GenerateGradebookEntriesTests(ModuleStoreTestCase):
     def setUp(self):
 
         # Turn off the signalling mechanism temporarily
-        settings._wrapped.default_settings.FEATURES['SIGNAL_ON_SCORE_CHANGED'] = False
+        #settings._wrapped.default_settings.FEATURES['SIGNAL_ON_SCORE_CHANGED'] = False
```

Comment:
If we're commenting out this line is there a use case where we would re-enable it?  If so we should document that scenario as a comment.  If not we should remove the line.